### PR TITLE
OIFS updates

### PR DESCRIPTION
**Pull request by seb-wahl**
Merge should be smooth for other models because modifications only affect OIFS, NEMO and XIOS, except for comments (`esm_plugins.yaml`) or three new variables (`oasis3mct.yaml`) that are only used if oasis3mct version 4.0 is used.

### CODE REVIEW

File: configs/components/oasis3mct/oasis3mct.yaml [hunk 1]
```diff
@@ -2,6 +2,9 @@ model: oasis3mct
 #model_dir: ${esm_master_dir}/oasis
 cf_name_dir: ${model_dir}
 time_step: 900 # changed by coupled setup anyway
+norestart: F
```
Comment (pgierz) [inline configs/components/oasis3mct/oasis3mct.yaml:5]: Should this be a string? I'm not sure if F just becomes False with the parser

File: configs/esm_software/esm_runscripts/esm_plugins.yaml [hunk 2]
```diff
@@ -26,7 +26,15 @@ core:
 
 # To add your custom plugin, see oifs.yaml, section compute_recipe, and compare to the list above.
 # the custom plugin 'preprocess' listed in the compute_recipe in oifs.yaml needs to be installed beforehand with
+#
 # git clone https://github.com/esm-tools-plugins/preprocess
 # cd preprocess
-# if required activate python3 e.g. module load anaconda3/bleeding_edge on mistral
+# if required activate python3 e.g. 
+# module load anaconda3/bleeding_edge on
 # pip install --user
```
Comment (pgierz) [inline configs/esm_software/esm_runscripts/esm_plugins.yaml:34]: You could add that to the list of commands oifs executes when it is installed.

**Review by pgierz — COMMENTED** [review]
I can't comment on all the yaml files for correctness, but it looks good. If it works we can put it in.

File: configs/esm_software/esm_runscripts/esm_plugins.yaml (hunk 2 above)
Comment (seb-wahl) [reply configs/esm_software/esm_runscripts/esm_plugins.yaml:34]: Good point, but I'll do that on a later stage given the fact that we are very close to the release

Comment (pgierz) [reply configs/esm_software/esm_runscripts/esm_plugins.yaml:34]: OK, I'll make an issue so we don't forget later on

**Pull request merged**